src/main/java/handler/LoginHandler.java

```diff
@@ -0,0 +1,57 @@
+package handler;
+
+import db.Database;
+import http.HttpRequest;
+import http.HttpResponse;
+import http.HttpResponseManager;
+import model.User;
+import sessions.Session;
+
+import java.util.Map;
+import java.util.Random;
+
+public class LoginHandler implements Action {
+    HttpResponseManager httpResponseManager;
+    @Override
+    public HttpResponse action(HttpRequest request) {
```

Comment:
```suggestion
    public HttpResponse response(HttpRequest request) {
```

이름 짓기 어려운데 명사보다는 적당한 동사가 좋겠네요. gpt에게 물어보면 어때요?